.github/workflows/gradle.yml

```diff
@@ -0,0 +1,31 @@
+name: Automatic Testing - Gradle
+
+#The action will be triggered everytime a pull request is made that includes files in the src directory (master branch)
+on: 
+  pull_request:
+    branches: 
+      - master
+    paths:
+      - 'src/**'
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2 ##Copies the repository files in the remote machine
+    - name: Set up JDK 1.8
+      uses: actions/setup-java@v1 #Sets up the Java version (8)
+      with:
+        java-version: 1.8
```

Comment:
any specific reason we should use this version of java?

Reply:
Not really. I just went with default again. Is there any specific version of Java we should define? I tought we could keep this one, and then modify it if we need any functionality of a higher version. But do you think we should configure it to any specific one?

Reply:
ok, lets just default for now.